android/.gitignore

```diff
@@ -21,4 +21,4 @@ local.properties
 .project
 lint.xml
 .gradletasknamecache
-
+.settings/
```

Comment:
Which app creates this folder?

Reply:
This appeared back when I was working on my PR with an Android UI fix. Don't know exactly what caused it, but I assume it was some gradle build step?

Reply:
What's inside?

Reply:
Seems to be some preference files related to Eclipse (which I never used) and [JDTLS](https://github.com/eclipse-jdtls/eclipse.jdt.ls) (Eclipse's Java language server, which I use in my editor). We should probably keep it?